static/app/views/dashboards/manage/dashboardGrid.tsx

```diff
@@ -101,6 +102,14 @@ function DashboardGrid({
     }
   }
 
+  async function handleFavorite(dashboard: DashboardListItem, isFavorited: boolean) {
+    try {
+      await updateDashboardFavorite(api, organization.slug, dashboard.id, isFavorited);
+    } catch (error) {
+      throw error;
+    }
+  }
```

Comment:
adding this function since we would need to call onDashboardsChange() so that the favourited dashboard is added to the 'My Favourites' section on the manage page